pkg/extensions/minimal.go

```diff
@@ -0,0 +1,23 @@
+// +build minimal
+
+package extensions
+
+import (
+	"time"
+
+	"github.com/anuvu/zot/pkg/log"
+	"github.com/anuvu/zot/pkg/storage"
+	"github.com/gorilla/mux"
+)
+
+// DownloadTrivyDB ...
+func DownloadTrivyDB(dbDir string, log log.Logger, updateInterval time.Duration) error {
+	return nil
+}
+
+func EnableExtension(extension *ExtensionConfig, log log.Logger, rootDir string) {
+	log.Info().Msg("given zot binary doesn't support any extensions, please build zot full binary for this feature")
```

Comment:
Good log msg.

Reply:
Yes, helpful - maybe also this should be a `log.Warn()` so it'll stand out a bit?

Reply:
Sure, I think its good to log as a warning because this part of code will run when user starts server providing extension configuration.

Reply:
Changed log message also, please review.